gpiozero/spi_devices.py

```diff
@@ -92,9 +131,9 @@ def _read(self):
     def value(self):
         """
         The current value read from the device, scaled to a value between 0 and
-        1.
+        1 (or -1 to +1 for certain devices operating in differential mode).
```

Comment:
I was mainly asking from a sourcetools PoV, where you need to know what range of `values` you can expect...
With the code currently you'd need to scale the (potential) -1 -> +1 range down to 0 -> 1 if connecting it to e.g. a PWMLED, and that in turn means the LED would never be below 50% brightness if used with a non-differential ADC.

Given the audience that GpioZero is aimed at, I wonder if any of the differential ADC code will actually *ever* get used??

Hmm, that's just given me another idea... I wonder if there could be some kind of `metavalue` property, that describes what type / range the value and values properties return, so that the sourcetools functions could check that things are 'hooked up correctly' and possibly auto-scale where appropriate? ;)  (Worth opening an issue for further discussion? Or is it just overcomplicating things)